### ISSUE TITLE
Backend Preview of IRRE-Elements does not filter deleted elements

### ISSUE THREAD

**Issue by sascha-schieferdecker** [Gernott/mask#26]
I've created an Element with an IRRE repeatable section. If a user deletes an IRRE element it doesn't show up in the frontend as expected. But the collection for the backend preview does still contain the deleted elements. 

Example:

```
<f:for each="{data.tx_mask_teaserbereich}" as="data_item" iteration="it">
...
</f:for>

```
The `tx_mask_teaserbereich` collection does contain the deleted elements. I think this behaviour should be changed if possible, so that the result is as expected: deleted elements should not be shown.

**Comment by sascha-schieferdecker** [comment]
Workaround for now using the VHS viewhelper extension:

```
<f:for each="{data.tx_mask_teaserbereich -> v:iterator.filter(propertyName: 'deleted', filter: 0, invert: 0)}" as="data_item" iteration="it">
...
</f:for>
```

**Comment by butu** [comment]
Fixed in 2.1.0
